frontend/appflowy_flutter/lib/workspace/presentation/settings/pages/settings_shortcuts_view.dart

```diff
@@ -460,6 +477,64 @@ class _ShortcutSettingTileState extends State<ShortcutSettingTile> {
   }
 }
 
+class EditShortcutBtn extends StatelessWidget {
+  const EditShortcutBtn({super.key, required this.onEdit});
+
+  final VoidCallback onEdit;
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: onEdit,
+      child: MouseRegion(
+        cursor: SystemMouseCursors.click,
+        child: FlowyTooltip(
+          message: LocaleKeys.settings_shortcutsPage_editTooltip.tr(),
+          child: const FlowySvg(
+            FlowySvgs.edit_s,
+            size: Size.square(16),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class ResetShortcutBtn extends StatelessWidget {
```

Comment:
For widgets used only in this file, we can make them private.
```suggestion
class _ResetShortcutButton extends StatelessWidget {
```